### ISSUE TITLE
Move to master branch

### ISSUE THREAD

**Issue by Shane32** [graphql-dotnet/server#417]
I think we should merge `develop` onto `master`, and continue on with the `master` branch, retaining the `develop` branch for preview versions.  `master` should be set as default in the repo if we make this change.  Note that this would be a normal merge, not squash and merge of course, to retain proper history.

**Comment by sungam3r** [comment]
But this means that we will not be able to build version 3.2, only 4.0 .

**Comment by Shane32** [comment]
You mean version 3.5?  (This is the `server` repo.)  Well the `master` branch was already updated for the 4.0 release when it was published, so I'm not sure what benefit it has over the `develop` branch.  Note that we can issue a release for any version on any branch.  I will be issuing a patch for 3.4 and publish 3.4.1 - see PR #382 (requires a cherry pick of PR #391 also).  But it can be done on any branch.

If we don't make `master` the current branch, I think we should force-push `master` back down to the 3.4 release.  Otherwise `master` serves no purpose.

**Comment by sungam3r** [comment]
Of course I confused two repos. Let's merge and make master as default.

**Comment by Shane32** [comment]
Ok I'll take care of it.

**Comment by sungam3r** [comment]
Thank you.

**Comment by Shane32** [comment]
@sungam3r `master` is default, and `develop` currently points to the same head.  All PRs were updated to have `master` as the base branch.

Shall I publish a 4.1.0 ?  Do you want to?

Also I created `master34` which contains 3.4 + #199 + GitHub Actions.  Since 3.4 relies on .Net Framework testing, the GitHub Actions had to be modified to run on Windows.  This broke the variables references, probably because it's not running on bash or something.  I didn't want to risk it publishing with the wrong version number so I commented out the stuff in the `publish.yml` that pulls the version number from the release and just hard-coded it in there.  After publishing 3.4.1, I bumped the version number there and in the Directory.Build.props to 3.4.2 and 3.4.2-preview respectively.  Hopefully we won't be issuing any more patches for 3.4 anyway.  Also noteworthy is that I had to delete the global.json file which was locking onto .Net SDK 2.1.  Since the 3.1 SDK should build dlls that are compatible with 2.1 / 2.2, I am using the 3.1 SDK to build and test in the workflows.  Finally, the test workflow runs on push rather than pull_request, so we can push commits directly to master34 and see results instead of requiring a PR.  It would still run on a PR also.  Oh, I did download and inspect the nupkg file with NuGet Package Explorer; everything seemed in order.  Last comment, I promise: if we delete `master34`, it shouldn't matter, since the tag or release or even the GitHub SHA can be used to create another branch if necessary.

**Comment by sungam3r** [comment]
> Shall I publish a 4.1.0 ? Do you want to?

Yes.

> Also noteworthy is that I had to delete the global.json file which was locking onto .Net SDK 2.1.

Good.

> if we delete master34

I'm 100% OK to keep only master and develop branches and go on from 4.1.

**Comment by Shane32** [comment]
First I deleted tags 4.0.0 and 4.0.2 since they do not match any nuget release.  The 4.0.1 tag is still there (without a matching "release") so it lines up to the nuget package version.

Then I pushed 4.0.2 which has the correct reference to 3.0.0, and then pushed 4.1.0 which references 3.1.0.  We should be good.  The 4.0.2 release should really have some comments in there about what's changed since 3.4.x, but I didn't spend any time on it.  Feel free to edit the release notes as you see fit.
